cerberus/kubernetes/client.py

```diff
@@ -205,10 +207,44 @@ def check_master_taint(master_nodes):
 
 
 # See if url is available
-def is_url_available(url):
-    requests.packages.urllib3.disable_warnings(category=InsecureRequestWarning)
-    response = requests.get(url, verify=False)
+def is_url_available(url, header=None):
+    response = requests.get(url, headers=header, verify=False)
     if response.status_code != 200:
         return False
     else:
         return True
+
+
+# Get bearer token
+def get_bearer_token():
+    try:
+        token = runcommand.invoke("oc whoami -t")
+        return token
+    except Exception as e:
+        logging.error("Exception when calling \
+                      Get Bearer Token: %s\n" % e)
+
+
+# Get routes for a specific namespaces
+def get_routes_from_namespace(namespace):
+    routes = runcommand.invoke('oc get routes -o yaml -n ' + namespace)
```

Comment:
s/oc/kubectl

Reply:
Routes is specific to openshift, my bad. Maybe run this only when the distribution is OpenShift similar to https://github.com/openshift-scale/cerberus/blob/master/start_cerberus.py#L219?

Reply:
I was able to do `kubectl get routes --all-namespaces` and get the same output as oc. Should I leave it as kubectl or oc?

Reply:
Routes resource won't be available on plain Kubernetes, let's use oc command and make this enhancement specific to openshift.